src/main/java/de/tum/in/www1/artemis/web/rest/LearningPathResource.java

```diff
@@ -108,16 +110,16 @@ public LearningPathResource(CourseService courseService, CourseRepository course
     @PutMapping("courses/{courseId}/learning-paths/enable")
     @FeatureToggle(Feature.LearningPaths)
     @EnforceAtLeastInstructorInCourse
-    public ResponseEntity<Void> enableLearningPathsForCourse(@PathVariable long courseId) {
+    public ResponseEntity<LearningPathsConfigurationDTO> enableLearningPathsForCourse(@PathVariable long courseId) {
```

Comment:
why is a Config returned here and not still void?

Reply:
So we can directly edit it in the client, especially if we reenable it and there were already changes beforehand